thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2145,13 +2145,13 @@ bool CJobMaster::fireException(IException *e)
     {
         case tea_warning:
         {
-            LOG(MCwarning, e);
+            LOG(MCprogress, e);
```

Comment:
We should kill MCerror, MCwarning and MCprogress, e.g use MCuserProgress - so it is clear what the target audience is.

Reply:
I will go through there uses in Thor and transition them